playbooks/roles/redis-exporter/defaults/main.yml

```diff
@@ -0,0 +1,20 @@
+---
+# The basic auth password to access the redis exporter.
+REDIS_EXPORTER_PASSWORD: null
+
+REDIS_EXPORTER_NGINX_CONFIG_PATH: /etc/nginx/sites-available/redis-exporter
+REDIS_EXPORTER_HTPASSWD_PATH: /etc/nginx/htpasswd
+
+REDIS_EXPORTER_USE_SSL: true
+
+REDIS_EXPORTER_REDIS_ADDR: 'redis://localhost:6379'
+REDIS_EXPORTER_CHECK_KEYS: 'huey.redis.opencraft'
```

Comment:
@shimulch, @0x29a, don't hard-code this here. It should be configured using Ansible host_vars or group_vars that apply only to Ocim servers.

Reply:
@lgp171188 Removed. Now those are being passed as variable from [Playbook](https://github.com/open-craft/ansible-playbooks/pull/177/commits/116bfa7a889210bb633d092ee16aab2fb878858d#diff-ec9878bb813db60641651149cd4209d3). I think those variables will be set on ``secrets`` repository?